GitHubSearch/Services/Realm/RealmService+Repository.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  RealmService+Repository.swift
+//  GitHubSearch
+//
+//  Created by yaoxp on 2020/8/24.
+//  Copyright © 2020 Eugene Karambirov. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+// MARK: - Repository CRUD
+extension RealmService {
```

Comment:
What do you think, if we put that in `RepositoryService` instead of as extension to `RealmService`? Looks like it is a business logic and not just working with Realm.

The way I see it, we should use `RealmService` inside of `RepositoryService`, to call basic CRUD operations. The only difference will be on the lines where you call `write` – there will be `realmService.write`.

Reply:
Yes, it is business logic.Your view is correct
I am learning MVVM through your code. Your code is great, and I benefited a lot. Thank you very much。

Reply:
So, will you fix it, please?